### PR TITLE
Move ConfigApis, MailApis, NavigationApis, PeopleApis and SharingApis to the new React components

### PR DESCRIPTION
**Pull request by MarcinJuraszek**

### CODE REVIEW

**Comment by erinha** [comment]
<p>Analyzed commit id : 8bf4e10146812a82f7cfa55943f9dff83b346ec4</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: dfc0607449fdb6e0cb181418434dab4f952d2eee</p>

**Comment by erinha** [comment]
<p>Analyzed commit id : 231d6c6a04702fb3f1424e81a2420041ee683383</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: 802a94db2e7a6e77825ed03a92dca9ce538e93bc</p>

**Review by aeunms — APPROVED** [review]
LGTM!

File: apps/teams-test-app/src/components/ConfigAPIs.tsx [hunk 1]
```diff
@@ -3,130 +3,130 @@ import React, { ReactElement } from 'react';
 
 import { noHostSdkMsg } from '../App';
```
[the remark this answers is not in the document]
Comment (MarcinJuraszek) [reply apps/teams-test-app/src/components/ConfigAPIs.tsx:4]: I'll send another PR for that

**Pull request merged**